onnx/common/ir.h

```diff
@@ -972,9 +969,11 @@ struct Graph final {
         initializers_.end());
     initializer_names_.erase(
         std::remove(initializer_names_.begin(), initializer_names_.end(), name), initializer_names_.end());
-    if (initializer_to_offset_map_.count(name) > 0) {
-      initializer_node_->eraseOutput(initializer_to_offset_map_[name]);
-      initializer_to_offset_map_.erase(name);
```

Comment:
after erasing output of `initializer_node_`, the `initializer_to_offset_map_` should be updated accordingly. However, it requires a full traverse to update every element in the map, which makes maintaining a map even worse than a naive search-and-erase.